drivers/rotator/WandererRotatorLite.cpp

```diff
@@ -0,0 +1,442 @@
+
+#include "WandererRotatorLite.h"
+#include "indicom.h"
+#include "connectionplugins/connectionserial.h"
+#include <cmath>
+#include <cstring>
+#include <memory>
+#include <termios.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <sstream>
+#include <iostream>
+
+
+// We declare an auto pointer to WandererRotatorLite.
+static std::unique_ptr<WandererRotatorLite> RotatorLite(new WandererRotatorLite());
+
+WandererRotatorLite::WandererRotatorLite()
+{
+    setVersion(1, 0);
+    RI::SetCapability(ROTATOR_CAN_REVERSE|ROTATOR_CAN_SYNC|ROTATOR_CAN_ABORT);
+}
+bool WandererRotatorLite::initProperties()
+{
+    INDI::Rotator::initProperties();
+    setDefaultPollingPeriod(500);
+    serialConnection->setDefaultBaudRate(Connection::Serial::B_19200);
+    // Home Rotator
+    IUFillSwitch(&HomeRotatorS[0], "HOME", "Start", ISS_OFF);
+    IUFillSwitchVector(&HomeRotatorSP, HomeRotatorS, 1, getDeviceName(), "ROTATOR_HOME", "Homing", MAIN_CONTROL_TAB,
+                       IP_RW, ISR_ATMOST1, 0, IPS_IDLE);
+    IUFillSwitch(&HomeS[0], "SetHomeButton", "Set Current Position as Home", ISS_OFF);
+    IUFillSwitchVector(&HomeSP, HomeS, 1, getDeviceName(), "SetHome", "Set Home", MAIN_CONTROL_TAB,IP_RW, ISR_ATMOST1,5, IPS_IDLE);
+
+    // Backlash Compensation Value
+    IUFillNumber(&RotatorBacklashN[0], "ROTATOR_BACKLASH_VALUE", "Angle", "%.2f", 0, 2., 0.1, 0.5);
+    IUFillNumberVector(&RotatorBacklashNP, RotatorBacklashN, 1, getDeviceName(), "ROTATOR_BACKLASH_angle",
+                       "Backlash",
+                       MAIN_CONTROL_TAB, IP_RW, 5, IPS_OK);
```

Comment:
No need to define this property, you need to use SetCapability.. for example

RI::SetCapability(ROTATOR_CAN_ABORT | ROTATOR_CAN_HOME | ROTATOR_HAS_BACKLASH);

Then you don't need to override SYNC for example, so it won't be defined anyway, but you need to override the INDI::Rotator backlash functions.

Reply:
Fixed,thanks :)